Assets/Package/Runtime/Console/Commands/MultipleArgumentsCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace BroWar.Debugging.Console.Commands
```

Comment:
Dałeś okejkę i to zignorowałeś, przenieść samplowe implementacje (w tym wypadku tylko ta klasa) do folderu Examples/Scripts i popraw namespace. Nie powinniśmy mieć takich komend bezpośrednio w paczce

Reply:
Nie tyle zignorowałem, co źle zrozumiałem. Skrypty definiujące pozostałe examplowe komendy (np. TestCommand, chyba że nie jest examplowa, to my bad) są w Package/Runtime/Console/Commands. Przeniosę wieczorem.